apps/web/src/components/common/Button/Button.stories.tsx

```diff
@@ -0,0 +1,167 @@
+import React, { useState } from 'react';
+import { MessageCircle, Plus, Trash2 } from 'lucide-react';
+
+import { IconButton } from '@/components/common/Button/variants/IconButton';
+
+import { Button } from './Button';
+import { FloatingButton } from './variants';
+
+import type { Meta, StoryObj } from '@storybook/react';
```

Comment:
```suggestion
import type { Meta, StoryObj } from '@storybook/nextjs-vite';
```

패키지는 react가 아닌 nextjs-vite로 사용 부탁드립니다!